### PR TITLE
add arabic language

### PR DESCRIPTION
**Pull request by obeidahr**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Review by Salloom99 — APPROVED** [review]
I've read the translation from English to Arabic and it seems accurate to me.

File: README-AR.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+- [Contributing to Open Source in general](#contributing-to-open-source-in-general)
+- [Direct GitHub searches](#direct-github-searches)
+- [Mozilla's contributor ecosystem](#mozillas-contributor-ecosystem)
+- [Useful articles for new Open Source contributors](#useful-articles-for-new-open-source-contributors)
+- [Using Version Control](#using-version-control)
+- [Open Source books](#open-source-books)
+- [Open Source contribution initiatives](#open-source-contribution-initiatives)
+- [Open Source programs to participate in](#open-source-programs-to-participate-in)
+- [License](#license)
```
Comment (erictleung) [inline README-AR.md:47]: Please update these to match your changes.

```suggestion
- [Contributing to Open Source in general](#المساهمة-في-المصادر-المفتوحة-بشكل-عام)
- [Direct GitHub searches](#عمليات-البحث-المباشرة-على-github)
- [Mozilla's contributor ecosystem](#النظام-البيئي-للمساهمين-في-mozilla)
- [Useful articles for new Open Source contributors](#مقالات-مفيدة-للمساهمين-الجدد-في-المصادر-المفتوحة)
- [Using Version Control](#باستخدام-التحكم-في-الإصدار)
- [Open Source books](#كتب-مفتوحة-المصدر)
- [Open Source contribution initiatives](#مبادرات-مساهمة-المصدر-المفتوح)
- [Open Source programs to participate in](#برامج-مفتوحة-المصدر-للمشاركة-فيها)
- [License](#license)
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for making time to translate this! Just one quick update to the table of contents and then I can merge it. Thanks!

**Review by erictleung — APPROVED** [review]
LGTM! Thanks for taking the time to make this translation ✨

**Pull request merged**